lib/services/local/userpreferences_test.go

```diff
@@ -141,6 +164,7 @@ func TestUserPreferencesCRUD2(t *testing.T) {
 					Onboard: &userpreferencesv1.OnboardUserPreferences{
 						PreferredResources: []userpreferencesv1.Resource{userpreferencesv1.Resource_RESOURCE_KUBERNETES},
 					},
+					PinnedResources: pinned,
```

Comment:
this test case is for "updating all settings at once" but since we already updated to this `pinned` previously, it's not really meeting the test case?

Reply:
Each test creates a new user preferences service so any upserting to user prefs is unique by test case. Unless I'm mistaken